cmd/logs.go

```diff
@@ -27,12 +29,39 @@ func newLogsCommand(client *client.Client) *Command {
 	return cmd
 }
 
-func runLogs(ctx *cmdctx.CmdContext) error {
-	err := monitor.WatchLogs(ctx, ctx.Out, monitor.LogOptions{
-		AppName:    ctx.AppName,
-		VMID:       ctx.Config.GetString("instance"),
-		RegionCode: ctx.Config.GetString("region"),
-	})
+func runLogs(cc *cmdctx.CmdContext) error {
+	ctx := createCancellableContext()
+
+	client := cc.Client.API()
+
+	opts := &logs.LogOptions{
+		AppName:    cc.Config.GetString("app"),
```

Comment:
You might want to use `cc.AppName` instead, since the app name might be coming from `fly.toml`.